go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/knative/client-contrib
+
+go 1.13
```

Comment:
There must be no top-level go.mod but each plugin must be self-contained.

Reply:
OK, I will move the go.mod and vendor to the plugin folder.

Reply:
I have moved the vendor to the plugin/migration folder. Thanks!
fixed in the second commit: https://github.com/knative/client-contrib/pull/6/commits/e77b98b6cebdccc7a81fa81f9c508488611a2d3b